.gitignore

```diff
@@ -1,2 +1,4 @@
 build/
 node_modules/
+
+h/static/scripts/coverage
```

Comment:
sure. Typically I prefer to have a tests directory and an app directory and not comingle them the way we do as well - then let the tests coverage happily live/gitignored inside of the tests dir. But, I digress, I'll get it pulled up